elixir/apps/web/test/web/live/resources/show_test.exs

```diff
@@ -0,0 +1,166 @@
+    conn: conn
+  } do
+    assert live(conn, ~p"/#{account}/resources/#{resource}") ==
+             {:error,
+              {:redirect,
+               %{
+                 to: ~p"/#{account}/sign_in",
+                 flash: %{"error" => "You must log in to access this page."}
+               }}}
+  end
+
+  test "renders not found error when resource is deleted", %{
+    account: account,
+    resource: resource,
+    identity: identity,
+    conn: conn
+  } do
+    resource = Fixtures.Resources.delete_resource(resource)
+
+    assert_raise Web.LiveErrors.NotFoundError, fn ->
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources/#{resource}")
+    end
+  end
+
+  test "renders breadcrumbs item", %{
+    account: account,
+    resource: resource,
+    identity: identity,
+    conn: conn
+  } do
+    {:ok, _lv, html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources/#{resource}")
+
+    assert item = Floki.find(html, "[aria-label='Breadcrumb']")
+    breadcrumbs = String.trim(Floki.text(item))
+    assert breadcrumbs =~ "Resources"
+    assert breadcrumbs =~ resource.name
+  end
+
+  test "allows editing resource", %{
+    account: account,
+    resource: resource,
+    identity: identity,
+    conn: conn
+  } do
+    {:ok, lv, _html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources/#{resource}")
+
+    assert lv
+           |> element("a", "Edit Resource")
+           |> render_click() ==
+             {:error,
+              {:live_redirect, %{to: ~p"/#{account}/resources/#{resource}/edit", kind: :push}}}
+  end
+
+  test "renders resource details", %{
+    account: account,
+    actor: actor,
+    identity: identity,
+    resource: resource,
+    conn: conn
+  } do
+    {:ok, lv, _html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources/#{resource}")
+
+    table =
+      lv
+      |> element("#resource")
+      |> render()
+      |> vertical_table_to_map()
+
+    assert table["name"] =~ resource.name
+    assert table["address"] =~ resource.address
+    assert table["created"] =~ actor.name
+
+    for filter <- resource.filters do
+      assert String.downcase(table["traffic filtering rules"]) =~ Atom.to_string(filter.protocol)
+    end
+  end
+
+  test "renders gateways table", %{
+    account: account,
+    identity: identity,
+    group: group,
+    resource: resource,
+    conn: conn
+  } do
+    {:ok, lv, _html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources/#{resource}")
+
+    gateway_groups =
+      lv
+      |> element("#gateway_instance_groups")
+      |> render()
+      |> table_to_map()
+
+    for gateway_group <- gateway_groups do
+      assert gateway_group["name"] =~ group.name_prefix
+      # TODO: check that status is being rendered
+    end
+  end
+
+  # TODO: Uncomment this test once functionality is implemented
```

Comment:
One more TODO

Reply:
I think this TODO make sense to get in this PR.  I'll get this updated shortly.

Reply:
I've added the delete functionality to the `show` page and enabled this test.